src/main/java/com/apigateway/apigateway/config/ZuulFilterImpl.java

```diff
@@ -33,7 +41,10 @@ public int filterOrder() {
     @Override
     public boolean shouldFilter() {
         log.info("Entering ZuulFilterImpl.shouldFilter");
-        return !RequestContext.getCurrentContext().getRequest().getRequestURI().equals("/v1/authenticate");
+        log.info("shouldFilter = {} ", (!RequestContext.getCurrentContext().getRequest().getRequestURI().equals("/v1/authenticate"))
```

Comment:
not a good practice. Put it in a variable and then log it and return the same variable.

Repeating code is not good practise